Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: build run
 
-BIN := bin/gullak.bin
+BIN := /usr/local/bin/gullak.bin
```

Comment:
Can this be `bin/` ? `/usr/local` will pollute the host env.

Reply:
done, path back to `$LOCAL_DIR/bin/..`